php/Dockerfile

```diff
@@ -8,85 +8,85 @@ ENV TERM dumb
 RUN apt-get update && apt-get upgrade -y
 
 RUN apt-get install -y \
+    # Included applications
     git \
     mysql-client \
     php-pear \
-    tree
+    tree \
```

Comment:
The `tree` utility package will be removed once the code that uses PHP's SPL iterators to do the same thing is merged into master.